cpg-console/src/main/kotlin/de/fraunhofer/aisec/cpg/console/Extensions.kt

```diff
@@ -253,7 +252,7 @@ fun getFanciesFor(original: Node, node: Node): List<Pair<AttributedStyle, Region
             fancyWord("new", node, list, styles.keyword)
 
             // check for primitive types
-            for (primitive in TypeParser.PRIMITIVES) {
+            for (primitive in node.language?.primitiveTypes?.toMutableList() ?: listOf()) {
```

Comment:
We can make this non mutable lists, its not important issue, but I would change this alongside, if we are touching this PR to fix other stuff